### PR TITLE
feat: improve style/wording of  identity request.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#574

### CODE REVIEW

File: src/components/identity/IdentityLocRequest.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.IdentityLocRequest .SelectLegalOfficer {
+    margin: 10px 20px 10px 10px;
+    text-align: left;
+    min-width: 420px;
+}
+
+.IdentityLocRequest .SelectLegalOfficer .drop-down {
```
Comment (gdethier) [inline src/components/identity/IdentityLocRequest.css:7]: This rule should be moved to `SelectedLegalOfficer.css`.

**Review by gdethier — APPROVED** [review]
Nice! One suggestion.

File: src/components/identity/IdentityLocRequest.css (hunk 1 above)
Comment (benoitdevos) [reply src/components/identity/IdentityLocRequest.css:7]: No. This breaks the current layout of Protection request, which also includes the componnent `SelectLegalOfficer`.

Comment (gdethier) [reply src/components/identity/IdentityLocRequest.css:7]: OK, but this is nonetheless a very bad practice (to have CSS styling specific to a child component defined in the stylesheet of a parent component). In that case, it is better, if possible, to provide some property to the component and let it apply conditionally some style in function of the property value.

Comment (benoitdevos) [reply src/components/identity/IdentityLocRequest.css:7]: Got it. This commit will be anyway removed, following the decision to convert the dialog into plain page.

**Pull request merged**